catalog/EDF/projects_EDF_MethaneSAT_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,297 @@
+// This is a generic template for an Earth Engine ee.ImageCollection dataset
+// STAC Collection entry. After replacing all the values, remove the
+// explanatory comments, but keep the "TODO" comment.
+//
+// This template covers the basic case with just one Jsonnet file defining
+// the dataset. For more complex examples with templating, look for directories
+// with `.libsonnet` files.
+//
+// See https://jsonnet.org/ for more documentation on the Jsonnet language used
+// here to create STAC JSON.
+//
+// Coding style:
+// - Indenting should be two spaces.
+// - Do not use tabs.
+// - Limit line length to around 80 characters unless it breaks URL strings.
+// - Use only ASCII characters. \n and 32..176 decimal range.
+// - Use American English.
+//
+// STAC Specifications:
+//   - Overall: https://github.com/radiantearth/stac-spec
+//   - Extensions: https://github.com/stac-extensions/stac-extensions.github.io
+//     - Electro-Optical: https://github.com/stac-extensions/eo
+//     - Synthetic Aperture Radar: https://github.com/stac-extensions/sar
+//     - Scientific: https://github.com/stac-extensions/scientific
+//     - Version: https://github.com/stac-extensions/version
+
+// This jsonnet file has to be named using the following convention:
+//   <subdir>/<id_with_slashes_replaced_with_underbars>.jsonnet.
+
+// The asset id as used in Earth Engine:
+//   ee.ImageCollection('TEMPLATE/IMAGE_COLLECTION_V2_3');
+local id = 'projects/EDF/MethaneSAT/public-preview/L3concentration';
```

Comment:
This id should point to an existing image collection, projects/EDF-MethaneSAT/assets/...something...

(if your Cloud project name is EDF-MethaneSAT)

Reply:
OK, thanks.  A fundamental misunderstanding that I had is that this an actual, literal cloud-project name.  I was under the impression that the project was more behind-the-scenes.  We're going to need to create a new project and I'll update the files to match.